cedar-policy-core/src/ast/policy.rs

```diff
@@ -618,6 +618,15 @@ impl From<&proto::LiteralPolicy> for LiteralPolicy {
     }
 }
 
+#[cfg(feature = "protobufs")]
+impl TryFrom<&proto::LiteralPolicy> for Policy {
+    type Error = ReificationError;
+    fn try_from(policy: &proto::LiteralPolicy) -> Result<Self, Self::Error> {
+        // TODO: do we need to provide a nonempty `templates` argument to `.reify()`
+        LiteralPolicy::from(policy).reify(&HashMap::new())
```

Comment:
Do we need an issue to track this work?

Reply:
There's a bunch of other code todos behind the `protobufs` feature flag, maybe we should have an issue to track all of them in general?  For instance a number of potential panics in this code

Reply:
Yes, we could have a single issue with all the "tasks" related to the feature.

Reply:
#1348